lib/workers/repository/updates/flatten.ts

```diff
@@ -216,8 +217,21 @@ export async function flattenUpdates(
       update.semanticCommits = semanticCommits;
     }
   }
-  return updates
-    .filter((update) => update.enabled)
+  let filteredOutCount = 0;
+  const filteredUpdates = updates
+    .filter((update) => {
+      if (update.enabled === false) {
```

Comment:
This original code assumes that `enabled` is always boolean and never missing. My new code does not assume that. I assume this wasn't a bug I'm fixing, but my new approach is more defensive